matrix/common/dss_zarr_store.py

```diff
@@ -46,15 +47,15 @@ class DSSZarrStore(MutableMapping):
     def _transform_key(self, key):
         return key.replace("/", self._separator_char)
 
-    def __init__(self, bundle_uuid, dss_instance="integration", bundle_version=None, replica="aws"):
+    def __init__(self, bundle_uuid, bundle_version=None, replica="aws"):
         """
-        :param dss_instance: name of dss environment such as 'dev' or 'staging'
         :param bundle_uuid: unique identifier for bundle in dss
         :param bundle_version: (optional) version tag for bundle in dss
         :param replica: (optional) "aws", "gcp", or "azure" to reflect dss cloud
         """
 
-        self._dss_client = self._get_dss_client(dss_instance)
+        dss_env = "integration" if os.environ['DEPLOYMENT_STAGE'] == "dev" else os.environ['DEPLOYMENT_STAGE']
```

Comment:
Could this fail a little more gracefully is `DEPLOYMENT_STAGE` isn't set? Maybe default to integration?